File: 0001_Two Sum/TwoSum.cpp
class Solution {
  public:
    vector<int> twoSum(vector<int>& nums,int target){
      unordered_map<int,int> record;
      for(int i = 0 ; i < num.size() ; i++)
      {
        int result = target - num[i];
        if(record.find(result) !+ record.end())
        {
          int res[] = {record[result],i};
          return vector<int>(res,res + 2);
        }
        record[nms[i]] = i;
      }
      return {};
    }
};

File: 0001_Two Sum/TwoSum.py
class Solution(object):
  def twoSum(self, nums, target):
    l = len(nums)
    print(nums)
    ans = []
    for i in range(l-1):
      for j in range(i+1,l):
        if nums[i] + nums[j] == target:
          ans.append(i)
          ans.append(j)
          print([i,j])
          break
  return ans
